src/core/configuration-component.ts

```diff
@@ -49,6 +55,17 @@ function initOptionChangedFunc(config, vueInstance: Pick<IVue, "$vnode" | "$prop
     setEmitOptionChangedFunc(config, vueInstance, innerChanges);
 }
 
+function getComponentInfo({name, isCollectionItem, ownerConfig }: Configuration, removed?: boolean): IComponentInfo {
+    const parentPath =  ownerConfig && ownerConfig.fullPath;
+    const optionPath = parentPath ? `${parentPath}.${name}` : name || "";
```

Comment:
`name || ""` indicates that name can be `undefined` or smth like, right?
In this case ``${parentPath}.${name}`` is equivalent to `${parentPath}.` - is this trailing dot OK?